subprojects/docs/src/docs/release/notes.md

```diff
@@ -138,6 +138,16 @@ The `CodeNarc` task also produces clickable links in failure messages to human-r
 
 The HTML report generated by `CodeNarc` has been updated to produce violation reports with sortable columns. 
 
+### Group opposite boolean task options together
+
+The console output of the `help` task renders options in alphabetical order.
+
+For better readability, this has been changed for opposite boolean task options.
+
+Disable options are now sorted after their opposite option, for example `--foo` and `--no-foo` are grouped together and are rendered after `--bar` and `--no-bar`.
```

Comment:
:lipstick: Let's use real option names instead of `foo` and `bar` in the documentation.